src/CMakeLists.txt

```diff
@@ -280,6 +283,7 @@ list(APPEND simulator_libs
 add_executable(Rover Rover.cpp)
 add_library(rover_common SHARED
   Autonomous.cpp
+  autonomous/AutonomousTask.cpp
```

Comment:
This file is already included in the `autonomous` library, don't think we need to put it here as well.